File: .py
class Student:
    def __init__(self, name):
        self.name = name

class Course:
    def __init__(self, title):
        self.title = title

class Registration:
    def __init__(self, student, course):
        self.student = student
        self.course = course

    def display(self):
        print(self.student.name, "registered for", self.course.title)


s = Student("Josh")
c = Course("Computer Programming")
r = Registration(s, c)
r.display()

File: .cpp
#include <iostream>
using namespace std;

class Student {
public:
    string name;
    Student(string n) { name = n; }
};

class Course {
public:
    string title;
    Course(string t) { title = t; }
};

class Registration {
    Student* student;
    Course* course;
public:
    Registration(Student* s, Course* c) {
        student = s;
        course = c;
    }

    void display() {
        cout << student->name << " registered for " << course->title;
    }
};

int main() {
    Student s("Josh");
    Course c("Computer Programming");
    Registration r(&s, &c);
    r.display();
    return 0;
}
